flink-python/src/main/java/org/apache/flink/python/env/beam/ProcessPythonEnvironmentManager.java

```diff
@@ -240,6 +240,10 @@ public String createRetrievalToken() throws IOException {
 		return env;
 	}
 
+	public void appendEnvironmentVariable(String key, String value) {
```

Comment:
Rename to `setEnvironmentVariable`. This method would replace the old value if key exists.